examples/curb-detection/lifelong_learning_bench/README.md

```diff
@@ -26,7 +26,7 @@ cd /ianvs #One might use another path preferred
 
 mkdir project
 cd project
-git clone https://github.com/kubeedge/ianvs.git   
+git clone -b feature-lifelong-n https://github.com/kubeedge/ianvs.git   
```

Comment:
In the formal version, the clone shall revised to the main branch

Reply:
Ok, I will revise it.